spec/iasm.dd

```diff
@@ -26,6 +26,38 @@ $(SPEC_S D x86 Inline Assembler,
         $(D D_InlineAsm_X86_64) version identifiers, respectively.
     )
 
+$(H2 $(LNAME2 asmstatements, Asm statement))
+
+$(GRAMMAR
+$(GNAME AsmStatement):
+    $(D asm) $(GLINK FunctionAttributes)$(OPT) $(D {) $(GLINK AsmInstructionList)$(OPT) $(D })
+
+    $(GNAME AsmInstructionList):
+        $(GLINK AsmInstruction) $(D ;)
+        $(GLINK AsmInstruction) $(D ;) $(I AsmInstructionList)
+    )
```

Comment:
Shouldn't this have the same order as here?
https://dlang.org/spec/statement.html#asm

Reply:
Actually i've copied this from grammar.dd.